_headers.ejs

```diff
@@ -9,6 +9,10 @@
 /serviceworker.js
   Cache-Control: no-cache
 
+# Module imports need CORS, so we enable it for the SDK file.
+/squoosh.mjs
+  Access-Control-Allow-Origin: *
+
```

Comment:
Add the header to all responses. May as well.

Reply:
This resource has a max-age of 31536000. Feels like it should be no-cache right?

Reply:
In general, what's the benefit of self-hosting this rather than providing it as a separate package? I'm worried about making changes to this resource that breaks people's sites.

Reply:
I thought that this way at least SDK and Squoosh stay in sync in terms of versioning. We can also not host it and publish it as a package, but not sure it’s that different in terms of breakage.